### ISSUE TITLE
Search result links broken for chat messages

### ISSUE THREAD

**Issue by nokome** [gitterHQ/gitter#1482]
When doing a search the links to matching chat messages are broken: they are missing the `owner/repo` part e.g. `https://gitter.im/?at=57266c30e10a59c06107a1d7` instead of `https://gitter.im/stencila/stencila?at=57266c30e10a59c06107a1d7`. Tested with Chrome on Ubuntu 51.0.2704.106

**Comment by MadLittleMods** [comment]
To clarify, the links are the dates at the top of each result,

![](http://i.imgur.com/tIijCFr.png)

**Comment by nokome** [comment]
@MadLittleMods : yep, I was referring to the link on the date. If you have a look at it is missing the `owner/repo` part e.g.

    <a class="room-item__search__link" href="https://gitter.im?at=5845d01cb4ffd59e38eccfe5">Dec 6th 9:37 AM</a>

When I submitted this bug report, for me that was causing the link to fail. Now, it works for me - presumably there's some JS highlighting the post when you click on the link. So you could close this if you like. But for semantics, you might want to fix the URL in any case.
